templates/account.html

```diff
@@ -35,7 +35,7 @@ <h1 class="u-no-margin">{{ username }}</h1>
   <section class="p-strip is-shallow">
     <div class="row">
       <div class="col-12">
-        <h3>My snaps</h3>
```

Comment:
@Lukewh Is this template used anywhere? I suppose it was replaced by `account-details` and `account-snaps` (?)